Extract&Load.py

```diff
@@ -0,0 +1,62 @@
+from sqlalchemy import create_engine
+from mysql_details import *
+import requests
+import pandas as pd
+import sys
+
+#Load data into Database
+def load(df):
+    try:
+        engine = create_engine("mysql+pymysql://{}:{}@{}/bikepoints".format(Username,Password,Host))
+        df.to_sql(name="bikepoints data", con=engine, index=False, if_exists="replace")
+        conn=engine.connect()
+        conn.execute("""ALTER TABLE `bikepoints data` 
+                        CHANGE COLUMN `ID` `ID` BIGINT NOT NULL PRIMARY KEY,
+                        CHANGE COLUMN `Name` `Name` TEXT NOT NULL;
+                        """)
+    except Exception as e:
+        print(e)
+
+
+#Extract Data from API
+def extract_load():
+    url = "https://api.tfl.gov.uk/bikepoint"
+    try:
+        response = requests.get(url)
+    except Exception as e:
+        print(e)
+        sys.exit()
+    data = response.json()
+
+    d = []
+    for bikepoint in data:
+        _id = bikepoint["id"][11:]

```

Comment:
avoid hard coded constants like '11' without explanation - makes it difficult to understand the reason behind this

Reply:
Well, in the bikepoint api all the "id" keys have value like "BikePoints_1" so i am only storing the integer part which starts from 11th index. Thanks for your suggestion I'll keep it in my mind and explain it there as well.